library.properties

```diff
@@ -1,5 +1,5 @@
 name=sensirion-sps
-version=0.0.6
+version=0.9.0
```

Comment:
Why 0.9.0? Don't we want to follow the version of the used sps30 driver (`SPS_DRV_VERSION_STR = "3.0.1"`) a bit? i.e. maybe using 0.3 or something?

Reply:
0.9 because I think we're closing in on API stability with 1.0.0 and it's proven reliable enough to warrant moving away from 0.1.0 - ACK to the reasoning not being particularly scientific. Regarding 3.0.1, I don't think it's worth the hassle to try to keep the version number in sync with the i2c driver as the features might differ.